File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 7/28/2025, 6:15:17 PM
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i1 = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[i1] = nums[i]
                i1 += 1
        return i1
        
        

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 8/2/2025, 4:31:18 PM
class Solution {
    public int removeElement(int[] nums, int val) {
        int k = 0;
        ArrayList <Integer> temp = new ArrayList <>();
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] != val) {
                temp.add(nums[i]);
            }
        }
        for (int i = 0; i < temp.size() ; i++) {
            nums[i] = temp.get(i);
        }
        return temp.size();
    }
}